portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -25,48 +25,68 @@
 import com.google.appengine.api.datastore.Entity;
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
 import com.google.gson.Gson;
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.Sentiment;
 
 import java.util.ArrayList;
+import java.util.List;
+
+import com.google.sps.servlets.Comment;
```

Comment:
nit: should this go with other `com.google....` imports on line 32 and above? All imports from the same source can be in one place... (just a thought)